spec/controllers/api/v1/recipes_controller_spec.rb

```diff
@@ -54,29 +55,50 @@
   describe '[PUT] #update' do
     let(:old_name) { 'Rosol' }
     let(:new_name) { 'Pierogi' }
-    let(:recipe) { create(:recipe, name: old_name) }
+    let(:old_price) { 5 }
+    let(:new_price) { 3 }
+    let(:recipe) { create(:recipe, name: old_name, price: old_price) }
 
     context 'with valid params' do
       it 'updates name' do
         expect do
           put :update, params: { id: recipe.id, recipe: { name: new_name } }
         end.to change { recipe.reload.name }.from(old_name).to(new_name)
       end
+      
+      it 'updates price' do
+        expect do
+          put :update, params: { id: recipe.id, recipe: { price: new_price } }
+        end.to change { recipe.reload.price }.from(old_price).to(new_price)
+      end
 
       it 'returns updated object' do
-        put :update, params: { id: recipe.id, recipe: { name: new_name } }
-        expect(JSON.parse(response.body)['name']).to eq(new_name)
+        put :update, params: { id: recipe.id, recipe: { name: new_name, price: new_price } }
+        recipeJSON = JSON.parse(response.body)
+        expect do 
+          recipeJSON['name'].to eq(new_name)
+        end
```

Comment:
These can be one-liners
```suggestion
        expect { recipeJSON['name'].to eq(new_name) }
```